governance/tag-governance.md

```diff
@@ -0,0 +1,541 @@
+- Subproject based initiatives MAY have different requirements that are outlined in the subproject
+  charter.
+
+### Roles
+
+ Assigned Tech Leads, Subproject Leads or TOC members serve as the general oversight of initiatives
+ depending on the parent governing body.
+
+### Formation
+
+There are three types of initiatives with different formation requirements, TAG initiatives,
+subproject initiatives, and TOC initiatives.
+
+#### TAG Initiatives
+
+General initiatives can be any "piece of work," such as white papers, reports, etc., that align
+with the scope of a TAG. The Tech Leads manage TAG Initiatives and MUST have at least one assigned
+as the sponsor to oversee the initiative's execution. They MAY delegate some of the responsibility
+of organizing and shepherding the initiative, but are still ultimately responsible for it.
+
+TAG initiatives can span multiple TAGs when something spans multiple tech domains, however one is
+ultimately assigned as the responsible group for execution and reporting. Initiatives that span
+many TAGs, or have no clear owner should be proposed as TOC initiatives. 
+
+#### Subproject Initiatives
+
+Subproject-based initiatives are defined in the subproject’s charter and are tailored to meet the
+subproject’s needs. They are essentially a customized GitHub issue template used to track the work
+of the subproject.
+
+#### TOC Initiatives
+
+The TOC may create and own initiatives when the scope of work spans the majority of TAGs or
+coordinate work around specific goals. In these instances, at least one TOC member MUST be assigned
+to organize, shepherd, and report on the initiative's status.
+
+## Technical Community Groups
+
+Technical Community Groups are a part of the broader [CNCF Community Groups Program]. They are
+topic and domain-focused or locality-based groups used as a rallying point for community members
+to discuss, share knowledge, and coordinate potential future initiatives. They are free to propose
+and shepherd initiatives when there is an explicit deliverable or objective they wish to work on.
+
+Community Groups have [much broader exposure], and have a suite of tools tailored for community
+management, making them the best place for discussion-oriented groups.
+
+Additionally, it is likely that future TAGs and subprojects will likely start as community groups.
+If they have proposed and completed numerous initiatives and cover a gap in the currently defined
+groups, then they may engage the TOC to determine if it makes sense to propose the formation of a
+TAG.
+
+
+[much broader exposure]: https://community.cncf.io/
+
+### Rules & Guidelines
+
+Adhere to the rules and guidelines outlined in the [Cloud Native Community Groups Program].
+
+### Formation
+
+Community Groups follow the formation process outlined in [Community Groups README].
+
+**NOTE:** Currently Technology focused groups do not have an issue template. These groups will
```

Comment:
`Currently Technology` lacks a definition or reference point

Reply:
Waiting on a PR to merge over there - Technical community group are slightly different, but it'll be updated when things merge.

Reply:
it's okay :)